src/savefilterdialog.cpp

```diff
@@ -49,7 +49,7 @@ bool SaveFilterDialog::SaveFilter(const QString& filename)
     qDebug() << "attempting to save...";
 
     // Validate filename
-    QRegularExpression regex("^[a-z0-9_]+$", QRegularExpression::CaseInsensitiveOption);
+    QRegularExpression regex("^[a-z0-9.\\-_ ]+$", QRegularExpression::CaseInsensitiveOption);
```

Comment:
Can we update the warning message below to match it as well? 
`             QStringLiteral("Only alphanumeric characters and underscores are allowed."));`
It can be more complicated to describe. Maybe `... characters, "-", "_", ".", or spaces ...`